extensions/himalaya/README.md

```diff
@@ -1,3 +1 @@
-# Himalaya
-
-For interacting with [Himalaya CLI](https://github.com/soywod/himalaya). Currently only acts on the default account.
+# XXXXXX
```

Comment:
Why did you change this?

Reply:
Ah, I mangled a commit. I created an extension from scratch, and ported over old things, and made a mistake here. I've fixed it now.